modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CsharpDotNet2ClientCodegen.java

```diff
@@ -57,8 +47,9 @@ public void processOpts() {
         super.processOpts();
 
         if (additionalProperties.containsKey(CLIENT_PACKAGE)) {
-            this.setClientPackage((String) additionalProperties.get(CLIENT_PACKAGE));
+            setClientPackage((String) additionalProperties.get(CLIENT_PACKAGE));
         } else {
+            setClientPackage(packageName + ".Client");
```

Comment:
Should this be done in the constructor, or the setClientPackage backed by a field with a default like others? If using this generator manually from code, we'll now have no default unless the user invokes processOpts.

If clientPackage has a sane default in the base type, please ignore. I haven't looked yet.

Reply:
You're right, just pushed new commit with initialization in constructor